flowmachine/flowmachine/core/server/query_proxy.py

```diff
@@ -72,6 +72,13 @@ class QueryProxyError(Exception):
     """
 
 
+class InvalidGeographyError(Exception):
+    """
+    Custom exception to indicate that the aggregation unit for a
+    geography query is invalid (corresponds to a 404 error in the API).
```

Comment:
Nice! I like the clear explanation in the docstring, including mentioning the 404 error.